MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include LICENSE
 
 graft websockets/py35
+include websockets/cext.c
```

Comment:
Well, let's just say I wanted to restart from fresh to avoid any Licence issues. Given how little code there's for that extension, there might not be an issue anyway.
The idea of doing an extension came after profiling python code on the server I'm working on. I stumbled upon #175 after that.